src/Pact/Native.hs

```diff
@@ -1587,6 +1589,26 @@ poseidonHackAChainDef = defGasRNative
         return $ toTerm $ poseidon intArgs
      | otherwise = argsError i as
 
+keccak256Def :: NativeDef
```

Comment:
We would need to add some translations for the FV system, see https://github.com/kadena-io/pact/blob/34f59718d1333367052b6c04310adac338fae534/src-tool/Pact/Analyze/Translate.hs#L1001 for an example. I'm happy to hop on a call to work on this together or PR this into this branch.